[tensorpipe] Build failure

Package: tensorpipe[core,test]:arm64-osx -> 2022-03-16#4

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 761220d10 2023-06-30 (15 hours ago)

**To Reproduce**

`vcpkg x-set-installed tensorpipe[core,test]`

**Failure logs**

```
-- Using cached tensorpipe-pr-435.patch.
-- Using cached pytorch-tensorpipe-52791a2fd214b2a9dc5759d36725909c1daa7f2e.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/pytorch-tensorpipe-52791a2fd214b2a9dc5759d36725909c1daa7f2e.tar.gz
-- Applying patch /Users/leanderSchulten/git_projekte/vcpkg/downloads/tensorpipe-pr-435.patch
-- Applying patch use-vcpkg.patch
-- Applying patch support-test.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean
-- Configuring arm64-osx
-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/tensorpipe/portfile.cmake:46 (vcpkg_cmake_install)
  scripts/ports.cmake:148 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/install-arm64-osx-dbg-out.log</summary>

```
[1/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/channel/error.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/channel/error.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/channel/error.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/channel/error.cc
[2/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/core/error.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/core/error.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/core/error.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/core/error.cc
[3/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/common/address.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/common/address.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/common/address.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/common/address.cc
[4/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/common/fd.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/common/fd.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/common/fd.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/common/fd.cc
...
Skipped 28 lines
...

[25/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/test_environment.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/test_environment.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/test_environment.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/test_environment.cc
[26/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/loop.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/loop.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/loop.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/transport/uv/loop.cc
[27/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/channel/mpt/channel_impl.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/channel/mpt/channel_impl.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/channel/mpt/channel_impl.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/channel/mpt/channel_impl.cc
[28/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/channel/mpt/factory.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/channel/mpt/factory.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/channel/mpt/factory.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/channel/mpt/factory.cc
[29/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/sockaddr.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/sockaddr.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/sockaddr.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/transport/uv/sockaddr.cc
[30/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/connection_impl.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/connection_impl.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/connection_impl.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/transport/uv/connection_impl.cc
[31/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/channel/mpt/context_impl.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/channel/mpt/context_impl.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/channel/mpt/context_impl.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/channel/mpt/context_impl.cc
[32/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/utility.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/utility.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/utility.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/transport/uv/utility.cc
[33/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/sockaddr_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/sockaddr_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/sockaddr_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/transport/uv/sockaddr_test.cc
[34/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/context_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/context_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/context_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/transport/context_test.cc
[35/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/loop_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/loop_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/loop_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/transport/uv/loop_test.cc
[36/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/listener_impl.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/listener_impl.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/listener_impl.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/transport/uv/listener_impl.cc
[37/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/uv_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/uv_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/uv_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/transport/uv/uv_test.cc
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/transport/uv/uv_test.cc:17:1: warning: 'InstantiateTestCase_P_IsDeprecated' is deprecated: INSTANTIATE_TEST_CASE_P is deprecated, please use INSTANTIATE_TEST_SUITE_P [-Wdeprecated-declarations]
INSTANTIATE_TEST_CASE_P(Uv, TransportTest, ::testing::Values(&helper));
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/gtest-param-test.h:538:38: note: expanded from macro 'INSTANTIATE_TEST_CASE_P'
  static_assert(::testing::internal::InstantiateTestCase_P_IsDeprecated(), \
                                     ^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-internal.h:1296:1: note: 'InstantiateTestCase_P_IsDeprecated' has been explicitly marked deprecated here
GTEST_INTERNAL_DEPRECATED(
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-port.h:2336:59: note: expanded from macro 'GTEST_INTERNAL_DEPRECATED'
#define GTEST_INTERNAL_DEPRECATED(message) __attribute__((deprecated(message)))
                                                          ^
1 warning generated.
[38/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/context_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/context_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/context_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/transport/uv/context_test.cc
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/transport/uv/context_test.cc:61:1: warning: 'InstantiateTestCase_P_IsDeprecated' is deprecated: INSTANTIATE_TEST_CASE_P is deprecated, please use INSTANTIATE_TEST_SUITE_P [-Wdeprecated-declarations]
INSTANTIATE_TEST_CASE_P(Uv, UVTransportContextTest, ::testing::Values(&helper));
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/gtest-param-test.h:538:38: note: expanded from macro 'INSTANTIATE_TEST_CASE_P'
  static_assert(::testing::internal::InstantiateTestCase_P_IsDeprecated(), \
                                     ^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-internal.h:1296:1: note: 'InstantiateTestCase_P_IsDeprecated' has been explicitly marked deprecated here
GTEST_INTERNAL_DEPRECATED(
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-port.h:2336:59: note: expanded from macro 'GTEST_INTERNAL_DEPRECATED'
#define GTEST_INTERNAL_DEPRECATED(message) __attribute__((deprecated(message)))
                                                          ^
1 warning generated.
[39/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/factory.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/factory.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/factory.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/transport/uv/factory.cc
[40/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/CMakeFiles/tensorpipe.dir/core/pipe_impl.cc.o -MF tensorpipe/CMakeFiles/tensorpipe.dir/core/pipe_impl.cc.o.d -o tensorpipe/CMakeFiles/tensorpipe.dir/core/pipe_impl.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/core/pipe_impl.cc
[41/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/connection_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/connection_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/connection_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/transport/uv/connection_test.cc
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/transport/uv/connection_test.cc:53:1: warning: 'InstantiateTestCase_P_IsDeprecated' is deprecated: INSTANTIATE_TEST_CASE_P is deprecated, please use INSTANTIATE_TEST_SUITE_P [-Wdeprecated-declarations]
INSTANTIATE_TEST_CASE_P(
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/gtest-param-test.h:538:38: note: expanded from macro 'INSTANTIATE_TEST_CASE_P'
  static_assert(::testing::internal::InstantiateTestCase_P_IsDeprecated(), \
                                     ^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-internal.h:1296:1: note: 'InstantiateTestCase_P_IsDeprecated' has been explicitly marked deprecated here
GTEST_INTERNAL_DEPRECATED(
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-port.h:2336:59: note: expanded from macro 'GTEST_INTERNAL_DEPRECATED'
#define GTEST_INTERNAL_DEPRECATED(message) __attribute__((deprecated(message)))
                                                          ^
1 warning generated.
[42/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/listener_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/listener_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/listener_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/transport/listener_test.cc
[43/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/connection_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/connection_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/connection_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/transport/connection_test.cc
[44/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/basic/basic_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/basic/basic_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/basic/basic_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/channel/basic/basic_test.cc
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/channel/basic/basic_test.cc:28:1: warning: 'InstantiateTestCase_P_IsDeprecated' is deprecated: INSTANTIATE_TEST_CASE_P is deprecated, please use INSTANTIATE_TEST_SUITE_P [-Wdeprecated-declarations]
INSTANTIATE_TEST_CASE_P(Basic, ChannelTestSuite, ::testing::Values(&helper));
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/gtest-param-test.h:538:38: note: expanded from macro 'INSTANTIATE_TEST_CASE_P'
  static_assert(::testing::internal::InstantiateTestCase_P_IsDeprecated(), \
                                     ^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-internal.h:1296:1: note: 'InstantiateTestCase_P_IsDeprecated' has been explicitly marked deprecated here
GTEST_INTERNAL_DEPRECATED(
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-port.h:2336:59: note: expanded from macro 'GTEST_INTERNAL_DEPRECATED'
#define GTEST_INTERNAL_DEPRECATED(message) __attribute__((deprecated(message)))
                                                          ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/channel/basic/basic_test.cc:30:1: warning: 'InstantiateTestCase_P_IsDeprecated' is deprecated: INSTANTIATE_TEST_CASE_P is deprecated, please use INSTANTIATE_TEST_SUITE_P [-Wdeprecated-declarations]
INSTANTIATE_TEST_CASE_P(Basic, CpuChannelTestSuite, ::testing::Values(&helper));
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/gtest-param-test.h:538:38: note: expanded from macro 'INSTANTIATE_TEST_CASE_P'
  static_assert(::testing::internal::InstantiateTestCase_P_IsDeprecated(), \
                                     ^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-internal.h:1296:1: note: 'InstantiateTestCase_P_IsDeprecated' has been explicitly marked deprecated here
GTEST_INTERNAL_DEPRECATED(
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-port.h:2336:59: note: expanded from macro 'GTEST_INTERNAL_DEPRECATED'
#define GTEST_INTERNAL_DEPRECATED(message) __attribute__((deprecated(message)))
                                                          ^
2 warnings generated.
[45/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/xth/xth_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/xth/xth_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/xth/xth_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/channel/xth/xth_test.cc
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/channel/xth/xth_test.cc:28:1: warning: 'InstantiateTestCase_P_IsDeprecated' is deprecated: INSTANTIATE_TEST_CASE_P is deprecated, please use INSTANTIATE_TEST_SUITE_P [-Wdeprecated-declarations]
INSTANTIATE_TEST_CASE_P(Xth, ChannelTestSuite, ::testing::Values(&helper));
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/gtest-param-test.h:538:38: note: expanded from macro 'INSTANTIATE_TEST_CASE_P'
  static_assert(::testing::internal::InstantiateTestCase_P_IsDeprecated(), \
                                     ^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-internal.h:1296:1: note: 'InstantiateTestCase_P_IsDeprecated' has been explicitly marked deprecated here
GTEST_INTERNAL_DEPRECATED(
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-port.h:2336:59: note: expanded from macro 'GTEST_INTERNAL_DEPRECATED'
#define GTEST_INTERNAL_DEPRECATED(message) __attribute__((deprecated(message)))
                                                          ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/channel/xth/xth_test.cc:30:1: warning: 'InstantiateTestCase_P_IsDeprecated' is deprecated: INSTANTIATE_TEST_CASE_P is deprecated, please use INSTANTIATE_TEST_SUITE_P [-Wdeprecated-declarations]
INSTANTIATE_TEST_CASE_P(Xth, CpuChannelTestSuite, ::testing::Values(&helper));
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/gtest-param-test.h:538:38: note: expanded from macro 'INSTANTIATE_TEST_CASE_P'
  static_assert(::testing::internal::InstantiateTestCase_P_IsDeprecated(), \
                                     ^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-internal.h:1296:1: note: 'InstantiateTestCase_P_IsDeprecated' has been explicitly marked deprecated here
GTEST_INTERNAL_DEPRECATED(
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-port.h:2336:59: note: expanded from macro 'GTEST_INTERNAL_DEPRECATED'
#define GTEST_INTERNAL_DEPRECATED(message) __attribute__((deprecated(message)))
                                                          ^
2 warnings generated.
[46/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/common/system_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/common/system_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/common/system_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/common/system_test.cc
[47/55] : && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake -E rm -f tensorpipe/libtensorpipe.a && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ar qc tensorpipe/libtensorpipe.a  tensorpipe/CMakeFiles/tensorpipe.dir/channel/error.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/channel/helpers.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/common/address.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/common/allocator.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/common/error.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/common/fd.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/common/socket.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/common/system.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/core/context.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/core/context_impl.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/core/error.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/core/listener.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/core/listener_impl.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/core/pipe.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/core/pipe_impl.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/transport/error.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/channel/basic/channel_impl.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/channel/basic/context_impl.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/channel/basic/factory.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/channel/xth/channel_impl.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/channel/xth/context_impl.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/channel/xth/factory.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/channel/mpt/channel_impl.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/channel/mpt/context_impl.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/channel/mpt/factory.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/connection_impl.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/context_impl.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/error.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/factory.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/listener_impl.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/loop.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/sockaddr.cc.o tensorpipe/CMakeFiles/tensorpipe.dir/transport/uv/utility.cc.o && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib tensorpipe/libtensorpipe.a && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake -E touch tensorpipe/libtensorpipe.a && :
[48/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/core/pipe_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/core/pipe_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/core/pipe_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/core/pipe_test.cc
[49/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/mpt/mpt_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/mpt/mpt_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/mpt/mpt_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/channel/mpt/mpt_test.cc
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/channel/mpt/mpt_test.cc:111:1: warning: 'InstantiateTestCase_P_IsDeprecated' is deprecated: INSTANTIATE_TEST_CASE_P is deprecated, please use INSTANTIATE_TEST_SUITE_P [-Wdeprecated-declarations]
INSTANTIATE_TEST_CASE_P(Mpt, ChannelTestSuite, ::testing::Values(&helper));
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/gtest-param-test.h:538:38: note: expanded from macro 'INSTANTIATE_TEST_CASE_P'
  static_assert(::testing::internal::InstantiateTestCase_P_IsDeprecated(), \
                                     ^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-internal.h:1296:1: note: 'InstantiateTestCase_P_IsDeprecated' has been explicitly marked deprecated here
GTEST_INTERNAL_DEPRECATED(
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-port.h:2336:59: note: expanded from macro 'GTEST_INTERNAL_DEPRECATED'
#define GTEST_INTERNAL_DEPRECATED(message) __attribute__((deprecated(message)))
                                                          ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/channel/mpt/mpt_test.cc:113:1: warning: 'InstantiateTestCase_P_IsDeprecated' is deprecated: INSTANTIATE_TEST_CASE_P is deprecated, please use INSTANTIATE_TEST_SUITE_P [-Wdeprecated-declarations]
INSTANTIATE_TEST_CASE_P(Mpt, CpuChannelTestSuite, ::testing::Values(&helper));
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/gtest-param-test.h:538:38: note: expanded from macro 'INSTANTIATE_TEST_CASE_P'
  static_assert(::testing::internal::InstantiateTestCase_P_IsDeprecated(), \
                                     ^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-internal.h:1296:1: note: 'InstantiateTestCase_P_IsDeprecated' has been explicitly marked deprecated here
GTEST_INTERNAL_DEPRECATED(
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-port.h:2336:59: note: expanded from macro 'GTEST_INTERNAL_DEPRECATED'
#define GTEST_INTERNAL_DEPRECATED(message) __attribute__((deprecated(message)))
                                                          ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/channel/mpt/mpt_test.cc:115:1: warning: 'InstantiateTestCase_P_IsDeprecated' is deprecated: INSTANTIATE_TEST_CASE_P is deprecated, please use INSTANTIATE_TEST_SUITE_P [-Wdeprecated-declarations]
INSTANTIATE_TEST_CASE_P(Mpt, MptChannelTestSuite, ::testing::Values(&helper));
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/gtest-param-test.h:538:38: note: expanded from macro 'INSTANTIATE_TEST_CASE_P'
  static_assert(::testing::internal::InstantiateTestCase_P_IsDeprecated(), \
                                     ^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-internal.h:1296:1: note: 'InstantiateTestCase_P_IsDeprecated' has been explicitly marked deprecated here
GTEST_INTERNAL_DEPRECATED(
^
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/gtest/internal/gtest-port.h:2336:59: note: expanded from macro 'GTEST_INTERNAL_DEPRECATED'
#define GTEST_INTERNAL_DEPRECATED(message) __attribute__((deprecated(message)))
                                                          ^
3 warnings generated.
[50/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/common/defs_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/common/defs_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/common/defs_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/common/defs_test.cc
[51/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/channel_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/channel_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/channel_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/channel/channel_test.cc
[52/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/channel_test_cpu.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/channel_test_cpu.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/channel_test_cpu.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/channel/channel_test_cpu.cc
[53/55] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -std=gnu++14 -MD -MT tensorpipe/test/CMakeFiles/tensorpipe_test.dir/core/context_test.cc.o -MF tensorpipe/test/CMakeFiles/tensorpipe_test.dir/core/context_test.cc.o.d -o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/core/context_test.cc.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/tensorpipe/src/9c1daa7f2e-806bb4005b.clean/tensorpipe/test/core/context_test.cc
[54/55] : && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  tensorpipe/test/CMakeFiles/tensorpipe_test.dir/test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/test_environment.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/context_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/connection_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/uv_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/context_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/loop_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/connection_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/sockaddr_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/listener_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/core/context_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/core/pipe_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/basic/basic_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/xth/xth_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/mpt/mpt_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/channel_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/channel_test_cpu.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/common/system_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/common/defs_test.cc.o -o tensorpipe/test/tensorpipe_test  tensorpipe/libtensorpipe.a  -luv_a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgmock.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/manual-link/libgtest_main.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libuv.a  -lpthread  -Xlinker -framework -Xlinker CoreFoundation  -Xlinker -framework -Xlinker IOKit  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgtest.a && :
FAILED: tensorpipe/test/tensorpipe_test 
: && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  tensorpipe/test/CMakeFiles/tensorpipe_test.dir/test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/test_environment.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/context_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/connection_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/uv_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/context_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/loop_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/connection_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/uv/sockaddr_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/transport/listener_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/core/context_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/core/pipe_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/basic/basic_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/xth/xth_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/mpt/mpt_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/channel_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/channel/channel_test_cpu.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/common/system_test.cc.o tensorpipe/test/CMakeFiles/tensorpipe_test.dir/common/defs_test.cc.o -o tensorpipe/test/tensorpipe_test  tensorpipe/libtensorpipe.a  -luv_a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgmock.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/manual-link/libgtest_main.a  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libuv.a  -lpthread  -Xlinker -framework -Xlinker CoreFoundation  -Xlinker -framework -Xlinker IOKit  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libgtest.a && :
ld: library not found for -luv_a
clang: error: linker command failed with exit code 1 (use -v to see invocation)
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "boost-uninstall",
      "default-features": false,
      "features": []
    }
  ],
  "$overrides": [
    {
      "name": "capstone",
      "version": "4.0.1-120373dc"
    }
  ],
  "$builtin-baseline": "338f99eed499b6df1628829e40695b396a70a1f7"
}

```
</details>

--- a/ports/tensorpipe/portfile.cmake
+++ b/ports/tensorpipe/portfile.cmake
@@ -40,7 +40,7 @@ vcpkg_cmake_configure(
 )
 vcpkg_cmake_install()
 vcpkg_copy_pdbs()
-vcpkg_cmake_config_fixup(CONFIG_PATH "share/cmake/Tensorpipe" PACKAGE_NAME "Tensorpipe")
+vcpkg_cmake_config_fixup(CONFIG_PATH "share/cmake/unofficial-tensorpipe" PACKAGE_NAME "unofficial-tensorpipe")
 
 vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.txt")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include"

--- a/ports/tensorpipe/portfile.cmake
+++ b/ports/tensorpipe/portfile.cmake
@@ -1,11 +1,5 @@
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
-vcpkg_download_distfile(INSTALL_PACKAGE_CONFIG_PATCH
-    URLS "https://patch-diff.githubusercontent.com/raw/pytorch/tensorpipe/pull/435.diff"
-    FILENAME tensorpipe-pr-435.patch
-    SHA512 7bcf604a967da36b8af936f8b8ab87b442834024b0b2cb886811c15e80893be842fbee2667bbbc39886814ec9b2f4ed0c2527de51fdb7dc293b25cce515f5e4b
-)
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO pytorch/tensorpipe
@@ -13,6 +7,7 @@ vcpkg_from_github(
     SHA512 1e5faf17a7236c5506c08cb28be16069b11bb929bbca64ed9745ce4277d46739186ab7d6597da7437d90ed2d166d4c37ef2f3bceabe8083ef3adbb0e8e5f227e
     PATCHES
         "${INSTALL_PACKAGE_CONFIG_PATCH}"
+        support-find-package.patch
         use-vcpkg.patch
         support-test.patch
 )

--- a/ports/tensorpipe/support-find-package.patch
+++ b/ports/tensorpipe/support-find-package.patch
@@ -1,14 +1,15 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 77df76d..97ab002 100644
+index 77df76d..bebb8c8 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -24,5 +24,16 @@ include(MiscCheck)
+@@ -24,5 +24,17 @@ include(MiscCheck)
  add_subdirectory(tensorpipe)
  
  install(EXPORT TensorpipeTargets
 -        DESTINATION share/cmake/Tensorpipe
 -        FILE TensorpipeTargets.cmake)
 +        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/unofficial-tensorpipe
++        NAMESPACE unofficial::tensorpipe::
 +        FILE unofficial-tensorpipe-targets.cmake)
 +
 +# Create TensorpipeConfig.cmake for find_package(unofficial-tensorpipe CONFIG)

--- a/ports/tensorpipe/support-find-package.patch
+++ b/ports/tensorpipe/support-find-package.patch
@@ -1,0 +1,56 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 77df76da7..284fa9928 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -24,5 +24,16 @@ include(MiscCheck)
+ add_subdirectory(tensorpipe)
+ 
+ install(EXPORT TensorpipeTargets
+-        DESTINATION share/cmake/Tensorpipe
++        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/Tensorpipe
+         FILE TensorpipeTargets.cmake)
++
++# Create TensorpipeConfig.cmake for find_package(Tensorpipe CONFIG)
++include(CMakePackageConfigHelpers)
++get_filename_component(CONFIG_FILE_PATH ${CMAKE_CURRENT_BINARY_DIR}/TensorpipeConfig.cmake ABSOLUTE)
++configure_package_config_file(
++  cmake/TensorpipeConfig.cmake.in ${CONFIG_FILE_PATH}
++  INSTALL_DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/Tensorpipe)
++
++# Install the generated config file
++install(FILES ${CONFIG_FILE_PATH}
++        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/Tensorpipe)
+diff --git a/cmake/TensorpipeConfig.cmake.in b/cmake/TensorpipeConfig.cmake.in
+new file mode 100644
+index 000000000..f2ea8ce47
+--- /dev/null
++++ b/cmake/TensorpipeConfig.cmake.in
+@@ -0,0 +1,15 @@
++@PACKAGE_INIT@
++include(CMakeFindDependencyMacro)
++
++find_dependency(libuv REQUIRED)
++
++get_filename_component(_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
++file(GLOB CONFIG_FILES "${_DIR}/TensorpipeConfig-*.cmake")
++foreach(f ${CONFIG_FILES})
++  include(${f})
++endforeach()
++
++# import targets 
++include("${_DIR}/TensorpipeTargets.cmake")
++
++check_required_components(@PROJECT_NAME@)
+diff --git a/tensorpipe/python/CMakeLists.txt b/tensorpipe/python/CMakeLists.txt
+index ce494722f..b722381f8 100644
+--- a/tensorpipe/python/CMakeLists.txt
++++ b/tensorpipe/python/CMakeLists.txt
+@@ -14,3 +14,8 @@ endif()
+ set(PYBIND11_CPP_STANDARD -std=c++14)
+ pybind11_add_module(pytensorpipe tensorpipe.cc)
+ target_link_libraries(pytensorpipe PRIVATE tensorpipe)
++
++install(TARGETS pytensorpipe
++        EXPORT TensorpipeTargets
++        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
++        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})

--- a/ports/tensorpipe/support-find-package.patch
+++ b/ports/tensorpipe/support-find-package.patch
@@ -1,5 +1,5 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 77df76da7..284fa9928 100644
+index 77df76d..97ab002 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
 @@ -24,5 +24,16 @@ include(MiscCheck)
@@ -7,24 +7,25 @@ index 77df76da7..284fa9928 100644
  
  install(EXPORT TensorpipeTargets
 -        DESTINATION share/cmake/Tensorpipe
-+        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/Tensorpipe
-         FILE TensorpipeTargets.cmake)
+-        FILE TensorpipeTargets.cmake)
++        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/unofficial-tensorpipe
++        FILE unofficial-tensorpipe-targets.cmake)
 +
-+# Create TensorpipeConfig.cmake for find_package(Tensorpipe CONFIG)
++# Create TensorpipeConfig.cmake for find_package(unofficial-tensorpipe CONFIG)
 +include(CMakePackageConfigHelpers)
-+get_filename_component(CONFIG_FILE_PATH ${CMAKE_CURRENT_BINARY_DIR}/TensorpipeConfig.cmake ABSOLUTE)
++get_filename_component(CONFIG_FILE_PATH ${CMAKE_CURRENT_BINARY_DIR}/unofficial-tensorpipe-config.cmake ABSOLUTE)
 +configure_package_config_file(
-+  cmake/TensorpipeConfig.cmake.in ${CONFIG_FILE_PATH}
-+  INSTALL_DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/Tensorpipe)
++  cmake/unofficial-tensorpipe-config.cmake.in ${CONFIG_FILE_PATH}
++  INSTALL_DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/unofficial-tensorpipe)
 +
 +# Install the generated config file
 +install(FILES ${CONFIG_FILE_PATH}
-+        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/Tensorpipe)
-diff --git a/cmake/TensorpipeConfig.cmake.in b/cmake/TensorpipeConfig.cmake.in
++        DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/unofficial-tensorpipe)
+diff --git a/cmake/unofficial-tensorpipe-config.cmake.in b/cmake/unofficial-tensorpipe-config.cmake.in
 new file mode 100644
 index 000000000..f2ea8ce47
 --- /dev/null
-+++ b/cmake/TensorpipeConfig.cmake.in
++++ b/cmake/unofficial-tensorpipe-config.cmake.in
 @@ -0,0 +1,15 @@
 +@PACKAGE_INIT@
 +include(CMakeFindDependencyMacro)
@@ -32,13 +33,13 @@ index 000000000..f2ea8ce47
 +find_dependency(libuv REQUIRED)
 +
 +get_filename_component(_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
-+file(GLOB CONFIG_FILES "${_DIR}/TensorpipeConfig-*.cmake")
++file(GLOB CONFIG_FILES "${_DIR}/unofficial-tensorpipe-config-*.cmake")
 +foreach(f ${CONFIG_FILES})
 +  include(${f})
 +endforeach()
 +
 +# import targets 
-+include("${_DIR}/TensorpipeTargets.cmake")
++include("${_DIR}/unofficial-tensorpipe-targets.cmake")
 +
 +check_required_components(@PROJECT_NAME@)
 diff --git a/tensorpipe/python/CMakeLists.txt b/tensorpipe/python/CMakeLists.txt

--- a/ports/tensorpipe/support-test.patch
+++ b/ports/tensorpipe/support-test.patch
@@ -26,10 +26,10 @@ index 53e8643..09baaa0 100644
 -  ${PROJECT_BINARY_DIR}/third_party/googletest EXCLUDE_FROM_ALL)
 +find_package(GTest CONFIG REQUIRED)
 +
-+if (TARGET uv)
-+    set(LIBUV_LIBRARY uv)
++if (TARGET libuv::uv)
++    set(LIBUV_LIBRARY libuv::uv)
 +else()
-+    set(LIBUV_LIBRARY uv_a)
++    set(LIBUV_LIBRARY libuv::uv_a)
 +endif()
  
  list(APPEND TP_TEST_LINK_LIBRARIES

--- a/ports/tensorpipe/vcpkg.json
+++ b/ports/tensorpipe/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "tensorpipe",
   "version-date": "2022-03-16",
-  "port-version": 4,
+  "port-version": 5,
   "description": "A tensor-aware point-to-point communication primitive for machine learning",
   "homepage": "https://github.com/pytorch/tensorpipe",
   "license": "BSD-3-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -7970,7 +7970,7 @@
     },
     "tensorpipe": {
       "baseline": "2022-03-16",
-      "port-version": 4
+      "port-version": 5
     },
     "termcolor": {
       "baseline": "2.1.0",

--- a/versions/t-/tensorpipe.json
+++ b/versions/t-/tensorpipe.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "1e5158bfc252016a7ca2e1d717dfd872eae280b1",
+      "version-date": "2022-03-16",
+      "port-version": 5
+    },
+    {
       "git-tree": "f470079788d337b13d5a7fb0b4f3663863f3c385",
       "version-date": "2022-03-16",
       "port-version": 4

--- a/versions/t-/tensorpipe.json
+++ b/versions/t-/tensorpipe.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "e21bac0d29d0acf55909818c619f0fd987fd82c5",
+      "git-tree": "5a085dcc97de3188e8fa82b123a7f3da7cd8de11",
       "version-date": "2022-03-16",
       "port-version": 5
     },

--- a/versions/t-/tensorpipe.json
+++ b/versions/t-/tensorpipe.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "1e5158bfc252016a7ca2e1d717dfd872eae280b1",
+      "git-tree": "4e5236156b95f2aa4659bba5ada4b6dcd548c52a",
       "version-date": "2022-03-16",
       "port-version": 5
     },

--- a/versions/t-/tensorpipe.json
+++ b/versions/t-/tensorpipe.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "4e5236156b95f2aa4659bba5ada4b6dcd548c52a",
+      "git-tree": "e21bac0d29d0acf55909818c619f0fd987fd82c5",
       "version-date": "2022-03-16",
       "port-version": 5
     },

